src/io/flutter/FlutterUtils.java

```diff
@@ -64,6 +67,16 @@ public static boolean exists(@Nullable VirtualFile file) {
     return file != null && file.exists();
   }
 
+  public static boolean isInFlutterProject(@NotNull PsiElement element) {
```

Comment:
Can we add some docs here (and in the other 'is flutter module/project' method) to say that this is the definitely-flutter-project method, and the other is the might-be-flutter one? That'll help the next person through :)

Reply:
Sure! 👍